lib/event_sourcery/event_processing/esp_process.rb

```diff
@@ -6,7 +6,7 @@ class ESPProcess
       def initialize(event_processor:,
                      event_source:,
                      subscription_master: EventStore::SignalHandlingSubscriptionMaster.new,
-                     after_fork:)
+                     after_fork: DEFAULT_AFTER_FORK)
         @event_processor = event_processor
         @event_source = event_source
         @subscription_master = subscription_master
```

Comment:
The `|| DEFAULT_AFTER_FORK` in the line below feels unnecessary, but omitting it breaks the ESPRunner.

Reply:
What do you think about using `after_fork: nil` in the constructor since we need the `|| DEFAULT_AFTER_FORK` anyway? That keeps the default value in one place.

Reply:
Yes, that would be fine too.